tgui/packages/tgui/interfaces/ClothingBooth/swatchForegroundShapes/spade.tsx

```diff
@@ -0,0 +1,23 @@
+import { SwatchForegroundProps } from "../type";
+
+export const SwatchSpade = (props: SwatchForegroundProps) => {
+  return (
+    <svg width="auto" height="100%" viewBox="0 0 6 6" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Rather than put the `props.color` on every child here, I *think* you can get away with putting `fill={props.color}` on the `svg` (and removing from each of the `path` children). If that doesn't work then fine as you have it, but would prefer that if we can.